projects/map/src/lib/helpers/ol-layer.helper.ts

```diff
@@ -59,12 +64,13 @@ export class OlLayerHelper {
     };
   }
 
-  public static createLayer(layer: LayerModel, projection: Projection, pixelRatio?: number): TileLayer<TileWMS> | ImageLayer<ImageWMS> | TileLayer<XYZ> | TileLayer<WMTS> | null {
+  public static createLayer(layer: LayerModel, projection: Projection, pixelRatio?: number,
```

Comment:
For readability I think it would be better to move all the arguments to a separate line instead of dividing in 2 lines